tests/framework/installer/ceph_installer.go

```diff
@@ -130,6 +130,29 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		return fmt.Errorf("Failed to create rook-operator pod : %v ", err)
 	}
 
+	csiRoles := h.Manifests.GetCSIRoles(namespace)
```

Comment:
what if we only added these artifacts only for the smoke test instead of all ceph integration test suites?

Reply:
it can be done in that way now, but i think in the future when we default to csi,  it makes sense to deploy the drivers in all tests.